demosplan/DemosPlanCoreBundle/Resources/config/services.yml

```diff
@@ -1352,5 +1355,8 @@ services:
 
     EDT\Wrapping\WrapperFactories\WrapperObjectFactory: '@demosplan\DemosPlanCoreBundle\Logic\EntityWrapperFactory'
 
+    Symfony\Component\Config\FileLocatorInterface:
+        alias: 'file_locator'
 
-
+    Symfony\Component\Routing\Loader\AnnotationClassLoader:
+        alias: 'routing.loader.annotation'
```

Comment:
Rewiring Symfony internals should be avoided. Not only is it bad practice, it can also lead to strange behaviours when Symfony updates it's service definitions to use more direct class injection.

Reply:
Without those we get errors like:
```  Cannot autowire service "demosplan\DemosPlanCoreBundle\Addon\AddonRoutingLo  
  ader": argument "$locator" of method "__construct()" references interface "  
  Symfony\Component\Config\FileLocatorInterface" but no such service exists.   
  You should maybe alias this interface to the existing "file_locator" servic  
  e.  ``` or ```Cannot autowire service "demosplan\DemosPlanCoreBundle\Addon\AddonRoutingLo  
  ader": argument "$loader" of method "__construct()" references class "Symfo  
  ny\Component\Routing\Loader\AnnotationClassLoader" but no such service exis  
  ts. You should maybe alias this class to the existing "routing.loader.annot  
  ation" service. ```

Reply:
Yes, correct. The recommended way to use aliased services is to declare their injections on the consuming side in the `services.yml`, see e.g. `demosplan/DemosPlanCoreBundle/Resources/config/services.yml:20`

Reply:
I adjusted the services.yml in [edd957a](https://github.com/demos-europe/demosplan-core/pull/958/commits/edd957af82168b6935d9e43a96917e0f48010596). Is this the way?